Assets/_Project/Code/Managers/SummaryManager.cs

```diff
@@ -9,12 +9,12 @@ public class SummaryManager : MonoBehaviour
     private void Start()
     {
         //foreach (Team team in listOfTeams)
-        foreach (TeamManager team in new TeamManager[] { new(), new(), new() })     // temp
+        foreach (TeamManager team in GameObject.Find("GameManager").GetComponent<GameManager>().Teams)     // temp
```

Comment:
Ten foreach jest dalej temporary? I jeżeli tak to dlaczego?

Reply:
Nie jest. Poprawiłem.